src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -715,6 +715,12 @@ public static class MsalError
         /// </summary>
         public const string TokenTypeMismatch = "token_type_mismatch";
 
+        /// <summary>
+        /// <para>What happens?</para>You are trying to use a feature which was marked as experimental
+        /// <para>Mitigation</para>When creating the PublicClientApplication or ConfidentialClientApplication, use .WithExperimentalFeatures() option. 
```

Comment:
See aka.ms/msal-net-experimental-features ?

Reply:
Is there anything else that we need to add in a wiki page? I feel that the 3 lines of comments pretty much explain the whole thing.

Reply:
yes, as discussed, a list of the current experimental features.